claim-db-worker/components/prismaSchemaEditor/InitOverlay.tsx

```
@@ -0,0 +1,30 @@
+export default function InitOverlay() {
+  return (
+    <div className="absolute inset-0 bg-[#181b21] flex items-center justify-center rounded-lg z-20 pointer-events-none">
+      <div className="text-center">
+        <svg
+          width="48"
+          height="60"
+          viewBox="0 0 58 72"
+          fill="none"
+          xmlns="http://www.w3.org/2000/svg"
+          className="mx-auto mb-4 animate-pulse"
+        >
```

Biome (2.1.2)

[error] Alternative text title element cannot be empty

For accessibility purposes, SVGs should have an alternative text, provided via title element. If the svg element has role="img", you should add the aria-label or aria-labelledby attribute.

(lint/a11y/noSvgWithoutTitle)